packages/react-mutation-mapper/src/component/gnomad/GnomadFrequency.tsx

```diff
@@ -145,7 +145,8 @@ export default class GnomadFrequency extends React.Component<GnomadFrequencyProp
             if (result["Total"].alleleFrequency === 0) {
                 display = <span>0</span>;
             } else {
-                display = <span>{parseFloat(result["Total"].alleleFrequency.toString()).toExponential(1)}</span>;
+                // show frequency as number with 4 significant digits
+                display = <span>{significantDigits(result['Total'].alleleFrequency, 4)}</span>;
```

Comment:
we have a library called Numeral.js (google it for docs) which should do this. lets use it instead.

Reply:
@alisman thanks! Basically what I'm looking for is to "round" number to 4 significant digits. I take a look at the Numeral.js doc but I'm not sure if it has the function I need, but I found another way to do it:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat
do you think it's ok to use?

Reply:
Ok, i see you are right numeral.js doesn't seem to have min/max significant digits.  I put another comment in.    As a general rule, try to avoid writing library code unless necessary and discuss with team if you think you do need to do so.